styles.css

```diff
@@ -0,0 +1,233 @@
+body {
+  font-family: 'Josefin Slab', serif;
+  font-size: 1.2rem;
+}
+
+/* Header */
+header {
+  background-image: url(images/header-image.jpg);
+  padding: 20px;
+}
+
+header img {
+  max-width: 45%;
+  height: auto;
+}
+
+header p {
+  color: green;
+  font-size: 1.5rem;
+  font-style: italic;
+  font-weight: bold;
+}
+
+.header-container {
+  display: flex;
+}
+
+.header-box1 {
+  flex: 1;
+}
+
+.header-box2 {
+  /* flex: 1; */
+  text-align: right;
+  display: flex;
+  flex-direction: column;
+}
+
+#checkout-icon {
+  color: white;
+  font-size: 1rem;
+}
+
+.checkout-page-link a {
+  color: white;
+  border: solid green;
+  background-color: green;
+  text-decoration: none;
+  padding: 4px;
+  margin-top: 8px;
+}
+
+.menu-bar {
+  background: lightgrey;
+  padding: 1px;
+}
+
+.menu-bar li {
+  display: inline;
+}
+
+.menu-bar a {
+  color: darkblue;
+  text-decoration: none;
+  font-weight: bold;
+}
+
+.menu-footer li {
+  display: inline;
+}
+
+.menu-footer a {
+  text-decoration: none;
+  font-weight: bold;
+  font-size: 1.05rem;
+}
+
+.homepage-article {
+  display: flex;
+}
+
+.ha-left {
+  flex: 1;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.ha-right {
+  flex: 2;
+  padding: 10px;
+}
+
+.ha-left img {
+  width: 90%;
+  min-width: 200px;
+  margin: 10px 0 0 10px;
+}
+
+.homepage-box-wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: 1.9rem;
+}
+
+.dog-box {
+  flex: 1 0 21%;
+  margin: 1rem;
+  border: 1px solid lightgrey;
+  min-width: 200px;
+}
+
+.dog-box img {
+  width: 100%;
+}
+
+.dog-box-detail {
+  padding: 10px;
+}
+
+.dog-box-detail button {
+  background-color: green;
+  font-weight: bold;
+  font-size: 0.875rem;
+  color: white;
+  width: 100%;
+  padding: 10px;
+}
+
+p .homepage-box-wrapper {
+  size: 0.1rem;
+}
+
+h4 {
+  color: darkblue;
+  font-weight: bold;
+}
+
+.see-more-dogs button {
+  background-color: lightgrey;
+  font-weight: bold;
+  font-size: 1.1rem;
+  color: darkblue;
+  padding: 10px;
+  display: block;
+  width: 200px;
+  margin: auto;
+}
+
+footer {
+  background-color: #003366;
+  color: white;
+  font-size: 0.9rem;
+  text-align: center;
+  margin-top: 40px;
+}
+
+footer img {
+  width: 200px;
+  height: 150px;
+}
+
+footer a {
+  color: white;
+  font-weight: bold;
+}
+
+/**Dogs Blog Checkout**/
+.blog1, .blog2, .blog3 {
+  display: flex;
+  margin: 1rem;
+}
+
+.blog-box-wrapper h3 {
+  margin: 1rem;
+  color: darkblue;
+  font-weight: bold;
+}
+
+.h-title {
```

Comment:
i like very semantic class names. what is the h?  Instead, think of blog-page-title or something...